types/vimeo__player/index.d.ts

```diff
@@ -135,17 +139,26 @@ export interface FullScreenChangeEvent {
     fullscreen: boolean;
 }
 
+export interface VimeoVideoQualityObject {
+    label: VimeoVideoQuality;
+    id: string;
```

Comment:
`id` might have a shape that could be defined by a string union literal, but it is not publicly documented (as far as I know), and so I've just defaulted to string.

Reply:
On further inspection, it doesn't seem that "label" cleanly fits the shape of `VimeoVideoQuality` either, and so I've loosened it to string as well.